### ISSUE TITLE
Accessibility review

### ISSUE THREAD

**Issue by serhalp** [serhalp/netlify-cache-inspector#115]
Review this app. Identify violations of the WCAG guidelines. Fix any identified issues. Make sure there's test coverage for a11y stuff. If there's canonical a11y tooling I should be using in this nuxt app (e.g. some sort of eslint plugin), install it, configure it appropriately, make sure it runs in CI (if necessary), and make it all green.
